### PR TITLE
Add Netatmo Thermostat

### PR DESCRIPTION
**Pull request by gieljnssns**
ref: https://github.com/home-assistant/home-assistant/pull/3888

### CODE REVIEW

**Comment by Landrash** [comment]
@gieljnssns No description?

File: source/_components/climate.netatmo.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+footer: true
+logo: netatmo.png
+ha_category: Climate
+ha_release: 0.31
+---
+
+
+The `netatmo` thermostat platform is consuming the information provided by a [Netatmo Smart Thermostat](https://www.netatmo.com/product/energy/thermostat) thermostat. This component allows you to view the current temperature and setpoint.
+
+To enable the Netatmo thermostat, you first have to set up [netatmo](/components/netatmo/), this will use discovery to add your camera.
```
Comment (jabesq) [inline source/_components/climate.netatmo.markdown]: "... add your camera." => wrong copy paste

File: source/_components/climate.netatmo.markdown [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: netatmo.png
+ha_category: Climate
+ha_release: 0.31
+---
+
+
```
Comment (jabesq) [inline source/_components/climate.netatmo.markdown]: Start this part with ### {% linkable_title Basic Configuration %}

File: source/_components/climate.netatmo.markdown [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+The `netatmo` thermostat platform is consuming the information provided by a [Netatmo Smart Thermostat](https://www.netatmo.com/product/energy/thermostat) thermostat. This component allows you to view the current temperature and setpoint.
+
+To enable the Netatmo thermostat, you first have to set up [netatmo](/components/netatmo/), this will use discovery to add your camera.
+
+If you want to select a specific thermostat, set discovery to False for [netatmo](/components/netatmo/) and add the following lines to your `configuration.yaml`:
```
Comment (jabesq) [inline source/_components/climate.netatmo.markdown]: Start this part with  ### {% linkable_title Advanced configuration %}

**Pull request merged**